operator/config/crd/bases/servicebroker.metabroker.suse.com_plans.yaml

```diff
@@ -45,8 +45,20 @@ spec:
                       binding credentials.
                     properties:
                       runContainer:
-                        description: The container specification for the logic of
-                          binding an Instance of the Plan.
+                        description: 'The container specification for the logic of
```

Comment:
I guess the output wasn't awesome. Ah well.

Reply:
This is generated.